contracts/test/integration/IntegrationTestVoltDeployment.t.sol

```diff
@@ -209,56 +209,56 @@ contract IntegrationTestVoltDeployment is DSTest, StdLib {
-        uint256 amountVoltIn = 100_000;
-        uint256 amountFeiOut = 101_000;
-
-        rariFEIPCVDeposit.deposit(); // get env cleaned up and ready for testing
-        vm.warp(28 days + block.timestamp);
-
-        uint256 startingUserVoltBalance = volt.balanceOf(address(this));
-        uint256 startingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
-
-        volt.approve(address(psm), amountVoltIn);
-        psm.redeem(address(this), amountVoltIn, amountFeiOut);
-
-        uint256 endingUserVoltBalance = volt.balanceOf(address(this));
-        uint256 endingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
-
-        assertEq(startingUserVoltBalance - endingUserVoltBalance, amountVoltIn);
-        assertEq(
-            startingPCVDepositFeiBalance - endingPCVDepositFeiBalance,
-            amountFeiOut
-        );
-    }
+    // /// this test uses FEI as the underlying asset and hooks into a FEI PCV Deposit
+    // function testMintAfterPriceIncrease() public {
+    //     uint256 amountFeiIn = 101_000;
+    //     uint256 amountVoltOut = 99_999;
+
+    //     rariFEIPCVDeposit.deposit(); // get env cleaned up and ready for testing
+    //     vm.warp(28 days + block.timestamp);
+
+    //     uint256 startingUserVoltBalance = volt.balanceOf(address(this));
+    //     uint256 startingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
+
+    //     fei.approve(address(psm), amountFeiIn);
+    //     psm.mint(address(this), amountFeiIn, amountVoltOut);
+
+    //     uint256 endingUserVoltBalance = volt.balanceOf(address(this));
+    //     uint256 endingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
+
+    //     assertEq(
+    //         endingUserVoltBalance - startingUserVoltBalance,
+    //         amountVoltOut
+    //     );
+    //     assertEq(
+    //         endingPCVDepositFeiBalance - startingPCVDepositFeiBalance,
+    //         amountFeiIn
+    //     );
+    // }
+
+    // /// this test uses FEI as the underlying asset and hooks into a FEI PCV Deposit
+    // function testRedeemAfterPriceIncrease() public {
+    //     uint256 amountVoltIn = 100_000;
+    //     uint256 amountFeiOut = 101_000;
+
+    //     rariFEIPCVDeposit.deposit(); // get env cleaned up and ready for testing
+    //     vm.warp(28 days + block.timestamp);
+
+    //     uint256 startingUserVoltBalance = volt.balanceOf(address(this));
+    //     uint256 startingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
+
+    //     volt.approve(address(psm), amountVoltIn);
+    //     psm.redeem(address(this), amountVoltIn, amountFeiOut);
+
+    //     uint256 endingUserVoltBalance = volt.balanceOf(address(this));
+    //     uint256 endingPCVDepositFeiBalance = rariFEIPCVDeposit.balance();
+
+    //     assertEq(startingUserVoltBalance - endingUserVoltBalance, amountVoltIn);
+    //     assertEq(
+    //         startingPCVDepositFeiBalance - endingPCVDepositFeiBalance,
+    //         amountFeiOut
+    //     );
+    // }
```

Comment:
@ElliotFriedman  this test has hardcoded values so will fail when changing block numbers, but if editing to variable values we'd essentially be replicating the PSM tests, so do we want to remove this, or change to replicate PSM tests here also?

Reply:
let's just remove these failing tests, they use code we don't use anymore and assume we are hooked into rari